docs/faq/basics/client-basics.md

```diff
@@ -63,4 +63,23 @@ use the cached message entity. Read more about it [here](xref:Guides.Concepts.Ev
 
 [MessageCacheSize]: xref:Discord.WebSocket.DiscordSocketConfig.MessageCacheSize
 [DiscordSocketConfig]: xref:Discord.WebSocket.DiscordSocketConfig
-[MessageUpdated]: xref:Discord.WebSocket.BaseSocketClient.MessageUpdated
\ No newline at end of file
+[MessageUpdated]: xref:Discord.WebSocket.BaseSocketClient.MessageUpdated
+
+## What is a shard/sharded client, and how is it different from the `DiscordSocketClient`?
+As your bot grows in popularity, it is recommended that you should section your bot off into separate processes.
+The [DiscordShardedClient] is essentially a class that allows you to easily create and manage multiple [DiscordSocketClient]
+instances, with each one serving a different amount of guilds.
+
+Aside from a few noteworthy differences in some event names, and specifying the amount of shards to connect with
+in a [DiscordSocketConfig], there are very few differences from the [DiscordSocketClient] class, and it is very straightforward
+to modify your existing code to use a [DiscordShardedClient] when necessary.
+
+The main differences are:
+1. You need to specify the total amount of shards, or shard ids, via [DiscordShardedClient]'s constructors.
+2. The [Connected], [Disconnected], [Ready], and [LatencyUpdated] events
```

Comment:
All `[]` references are not linked correctly. Add corresponding xrefs at the bottom of the section.

e.g.,

```md
2. The [Connected], [Disconnected], [Ready], and [LatencyUpdated] events
...
[Connected]: xref:Discord.WebSocket.DiscordSocketClient.Connected
```

---
Current Snapshot

![image](https://user-images.githubusercontent.com/5843208/47242060-45ce7380-d420-11e8-9259-68ec89fda1fc.png)